ArduPlane/Log.cpp

```diff
@@ -287,6 +326,36 @@ void Plane::Log_Write_Vehicle_Startup_Messages()
     gps.Write_AP_Logger_Log_Startup_messages();
 }
 
+/*
+  log a COMMAND_INT message
+ */
+void Plane::Log_Write_MavCmdI(const mavlink_command_int_t &mav_cmd)
```

Comment:
Logging these is quite valid, but we should actually be doing that as common logging up in the GCS_MAVLink level.

Reply:
discussed this with tridge, the idea of logging COMMAND_INT in AP_Vehicle instead of here, and he said he's ok with it going in the way it is for the moment, as doing AP_Vehicle is a much larger change that's not immediately in-scope for changes to a guided-slew-mode pr like this one. possible future enhancement.